pkg/metrics/metrics.go

```diff
@@ -58,6 +57,8 @@ type Config struct {
 	namespace string
 	// recorder to generate events
 	recorder record.EventRecorder
+	// operatorName is the name of the operator
+	operatorName string
```

Comment:
I don't think operatorName needs to be added to the struct. It does not seem to be part of `Config`.